docker-compose.yml

```diff
@@ -26,6 +26,11 @@ services:
       AWS_ACCESS_KEY_ID: longkey
       AWS_SECRET_ACCESS_KEY: verysecretkey
 
+  message-bus:
```

Comment:
I assume this should be private to Celery, as we don't write or read values from the DAGs code for example. Possibilities for a service name?
- `celery-broker` (not very precise because brokers usually have their own protocol like AMQP, this is just part of the broker concept)
- `celery-broker- backend` (matches `db` as that doesn't mention `postgres`, and Celery's [terminology](https://docs.celeryproject.org/en/latest/getting-started/first-steps-with-celery.html#keeping-results))
- `celery-broker-redis` (to communicate what this is)

Reply:
name updated to `airflow-broker`